detection_rules/rule.py

```diff
@@ -1199,6 +1238,16 @@ def post_conversion_validation(self, value: dict, **kwargs):
     def validate_remote(remote_validator: 'RemoteValidator', contents: 'TOMLRuleContents'):
         remote_validator.validate_rule(contents)
 
+    @classmethod
+    def from_rule_resource(
+            cls, rule: dict, creation_date: str = TIME_NOW, updated_date: str = TIME_NOW,
```

Comment:
Are there really no dates that we can use instead of generating new ones?

Reply:
in reality, if a user is reconciling updates rather than a net new pull, they would want to fill the contents _into_ an existing rule with existing meta (and dates)